### DISCUSSION TITLE
How to get user info on server side?

### DISCUSSION THREAD

**Discussion by danddinh** [ToolJet/ToolJet#6370] [Q&A] [answered]
As shown in `globals.currentUser`, there are some fields about user. Are these info sent from client only? or having the matching server-side variables?

eg. SQL query
`select * from notes as t where t.email={{globals.currentUser.email}}`

The value of email is sent from client or already existing on server side?

**Comment by tejakummarikuntla** [comment]
`globals.currentUser.email` brings up the email address of the current logged-in user; the user will be logged in only when they are added to your workspace, and the data is fetched to present @dandeancook.

Kindly elaborate on your query if you are looking for something else.

**Reply by danddinh** [reply]
> current logged-in user

The SQL query is done on server side, obviously, thus if `globals.currentUser.email` is altered before sending to server, in that way one user can get the data of another user (coz some of my queries rely on field `email` to get the data of exact user). Are there any security holes here?

**Comment by danddinh** [comment, marked as answer]
Tested, variables are sent from client to server instead of saving the variables of the same values on server side.

**Reply by yaronshanisima** [reply]
So there is a huge security hole here with this approach of filtering? So how can you solve it?
